test/unit/style-spec/fixture/bad-dasharray.output-api-supported.json

```diff
@@ -1,10 +1,10 @@
 [
   {
-    "line": 16,
-    "message": "layers[0].paint.line-dasharray[1]: -2 is less than the minimum value 0"
+    "message": "layers[0].paint.line-dasharray[1]: -2 is less than the minimum value 0",
```

Comment:
should this file have been changed?

Reply:
I bet these changes are related to using a different version of node.js to update fixtures.

Reply:
No strong preference here. Seems like a dependency switched their implementation or like someone switched to a `Map` since non-numeric keys should remain ordered consistently. Could manually un-reorder them to reduce git-blame noise, but doesn't seem like big deal either way.